geoapps/block_model_creation/application.py

```diff
@@ -0,0 +1,486 @@
+                            style={
+                                "width": "25%",
+                                "display": "inline-block",
+                                "margin-top": "20px",
+                            },
+                        ),
+                        dcc.Dropdown(
+                            id="objects",
+                            value=getattr(self.params.objects, "name", None),
+                            options=[
+                                {
+                                    "label": obj.parent.name + "/" + obj.name,
+                                    "value": obj.name,
+                                }
+                                for obj in self.params.geoh5.objects
+                            ],
+                            style={
+                                "width": "75%",
+                                "display": "inline-block",
+                                "margin_bottom": "20px",
+                            },
+                        ),
+                    ]
+                ),
+                html.Div(
+                    [
+                        dcc.Markdown(
+                            children="Lateral extent",
+                            style={"width": "25%", "display": "inline-block"},
+                        ),
+                        dcc.Dropdown(
+                            id="xy_reference",
+                            value=getattr(self.params.xy_reference, "name", None),
+                            options=[
+                                {
+                                    "label": obj.parent.name + "/" + obj.name,
+                                    "value": obj.name,
+                                }
+                                for obj in self.params.geoh5.objects
+                            ],
+                            style={
+                                "width": "75%",
+                                "display": "inline-block",
+                                "margin_bottom": "20px",
+                            },
+                        ),
+                    ]
+                ),
+                html.Div(
+                    [
+                        dcc.Markdown(
+                            children="Name",
+                            style={"width": "25%", "display": "inline-block"},
+                        ),
+                        dcc.Input(
+                            id="new_grid",
+                            value=self.params.new_grid,
+                            style={
+                                "width": "50%",
+                                "display": "inline-block",
+                                "margin_bottom": "20px",
+                            },
+                        ),
+                    ]
+                ),
+                html.Div(
+                    [
+                        dcc.Markdown(
+                            children="Minimum x cell size",
```

Comment:
Somehow the name get's displayed at the end?
![image](https://user-images.githubusercontent.com/55204635/180062364-6244db71-dc72-425b-b547-260c51ab7abc.png)